source/_integrations/alexa_devices.markdown

```diff
@@ -65,9 +65,42 @@ This means that the specified country may need a special setting.
 Open a issue with all details to investigate
 {% endnote %}
 
+## Actions
+
+### Available Actions
+
+Available actions: `notify.send_message`, `alexa_devices.send_sound`, `alexa_devices.send_text_command`
+
+#### Action `notify.send_message`
+
+Devices with appropriate functionality will have speak and announce notify entities created. These can be used as the target for the `notify.send_message` action.
+
+{% tip %}
+When sending notifications to multiple devices you may experience delays due to rate limiting by Amazon.  You can avoid this by sending notifications to speaker groups created in Alexa.
+{% endtip %}
+
+#### Action `alexa_devices.send_text_command`
+
+This action essentially allows you to control Alexa using text commands rather then speech. You should be able to request anything you would via speech using this action.
```

Comment:
```suggestion
This action essentially allows you to control Alexa using text commands rather than speech. You should be able to request anything you would via speech using this action.
```